### PR TITLE
[wxwidgets] Add feature fonts

### LINKED ISSUE #26065: [wxwidgets] Enable use of private fonts on Linux.
****

**Is your feature request related to a problem? Please describe.**
I need to be able to use the private font functionality of wxWidgets on Linux.  That functionality is not enabled by default.

**Proposed solution**
I'd like the ability to specify in my project's vcpkg manifest file that I need the private font functionality of wxWidgets.

**Describe alternatives you've considered**
I tried adding pango to my vcpkg manifest file since the private font functionality requires it on Linux.  The only other alternative would be to build wxWidgets myself outside of vcpkg, which defeats the purpose of vcpkg.

**Additional context**
The [wxWidgets documentation](https://docs.wxwidgets.org/3.2/classwx_font.html#ab96266bba385bc770a40ec4c7d1ec620) gives some more information about enabling this functionality.

### PR DESCRIPTION
**Pull request by Adela0814**
Fix https://github.com/microsoft/vcpkg/issues/26065

Add feature `fonts`.

All features tested on the following triplets:

- x64-windows
- x86-windows
- x64-windows-static

### CODE REVIEW

**Review by JonLiu1993 — DISMISSED** [review]

File: ports/wxwidgets/vcpkg.json [hunk 1]
```diff
@@ -42,6 +43,9 @@
     "example": {
       "description": "Example source code and CMake project"
     },
+    "fonts": {
+      "description": "Enable to use the font functionality of wxWidgets"
```
Comment (ras0219-msft) [inline ports/wxwidgets/vcpkg.json:47]:
```suggestion
      "description": "Enable to use the font functionality of wxWidgets",
      "dependencies": [
          { "name": "pango", "platform": "!windows & !osx" },
          { "name": "fontconfig", "platform": "!windows & !osx" }
      ]
```

Comment (ras0219-msft) [reply ports/wxwidgets/vcpkg.json:47]: On UNIX, this feature requires additional dependencies: https://github.com/wxWidgets/wxWidgets/blob/6268b5c619ac7bd7479c0912275bce89d67d1f23/build/cmake/init.cmake#L465-L472

**Review by ras0219-msft — DISMISSED** [review]
Thanks for the PR!

File: ports/wxwidgets/vcpkg.json (hunk 1 above)
Comment (Adela0814) [reply ports/wxwidgets/vcpkg.json]: Thanks for your comment, fixed.

Comment (dg0yt) [reply ports/wxwidgets/vcpkg.json]: Until vcpkg has gtk3 again, this is all moot. Port wxwidgets needs gtk3 on Linux, which is waiting for another CI image update. With system GTK3,  it pulls in system versions of many libs. And there is no testing in vcpkg CI.

**Review by JonLiu1993 — APPROVED** [review]

**Comment by JavierMatosD** [comment]
Thanks!

**Pull request merged**